strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenArtifactControllerTest.java

```diff
@@ -840,14 +840,18 @@ public void testMetadataAtGroupAndArtifactIdLevel(@MavenRepository(repositoryId
 
         MavenArtifactDeployer artifactDeployer = buildArtifactDeployer(Paths.get(System.getProperty("java.io.tmpdir")));
 
+        try{
```

Comment:
Cosmetic: Braces on new lines please. :)
```suggestion
        try
        {
```